backend/typescript/services/implementations/reviewService.ts

```diff
@@ -32,6 +37,156 @@ class ReviewService implements IReviewService {
     if (db !== sequelize) sequelize.close(); // Using test db instead of main db
   }
 
+  /* eslint-disable class-methods-use-this, no-await-in-loop */
+  async findOrCreateTag(
```

Comment:
nit: add function documentation to all functions in this file

Reply:
@obaranek This is important, we need to document this file well lol